projects/plugins/crm/src/js/components/automations-admin/components/edit-modal/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button } from '@automattic/jetpack-components';
+import { Modal } from '@wordpress/components';
+import { sprintf, __ } from '@wordpress/i18n';
+import { useCallback } from 'react';
+import { StepConfig } from '../step-config';
+import styles from './styles.module.scss';
+import type { Step, Workflow } from 'crm/state/automations-admin/types';
+
+type EditModalProps = {
+	workflow: Workflow;
+	isOpen: boolean;
+	onClose: () => void;
+};
+
+export const EditModal: React.FC< EditModalProps > = ( { isOpen, onClose, workflow } ) => {
+	const steps: Step[] = [];
+
+	let stepId: string | undefined = workflow.initial_step;
+	while ( stepId ) {
+		const step: Step = workflow.steps?.[ stepId ];
+		steps.push( step );
+		stepId = step.next_step;
+	}
+
+	const onSave = useCallback( () => {
+		// TODO: save workflow here
+		onClose();
+	}, [ onClose ] );
+
+	const onCancel = useCallback( () => {
+		// TODO: reload workflow here
+		onClose();
+	}, [ onClose ] );
+
+	return (
+		isOpen && (
+			<Modal onRequestClose={ () => onClose() }>
+				<div className={ styles.container }>
+					<h1>
+						{ sprintf(
+							/* translators: workflowName is a string which is used to identify the workflow */
+							__( 'Edit %(workflowName)s', 'zero-bs-crm' ),
+							{
+								workflowName: workflow.name,
+							}
+						) }
+					</h1>
+					<div className={ styles.subheader }>
+						{ __( 'Define and customize the workflow', 'zero-bs-crm' ) }
+					</div>
+					{ steps.map( step => (
+						<div className={ styles[ 'step-container' ] }>
+							<StepConfig workflowId={ workflow.id } step={ step } />
+						</div>
+					) ) }
+					<div className={ styles[ 'button-container' ] }>
```

Comment:
The spacing above and below the action buttons are the same in Figma

![Screenshot 2023-09-28 at 15 42 44](https://github.com/Automattic/jetpack/assets/3846700/770e0347-75a1-4eff-9a4f-1669fb0780d3)